PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -22,6 +24,8 @@
     private Button startButton;
     private Button newUserButton;
     private Button aboutButton;
+    private int alert_icon = R.drawable.ic_launcher;
```

Comment:
since API 22:
But if want it to be implemented in some other way we can use this:
ResourcesCompat.getDrawable() for unstyled drawable.
Should I go ahead and correct this.